lib/autoupdate/feature_ent.go

```diff
@@ -0,0 +1,26 @@
+//go:build webassets_ent
+// +build webassets_ent
```

Comment:
I'm not sure if this is a reliable way to detect ent builds. When developping we often do ent builds without webassets.

Reply:
this one was wrong way to identify the build version, replacing with `lib/modules`